index.js

```diff
@@ -19,6 +23,18 @@ const app = new Koa()
 
 const globalTags = GLOBAL_TAGS && GLOBAL_TAGS.split(',')
 
+var metricNameWhitelist = ['cloudflareError']
```

Comment:
```suggestion
let metricNameWhitelist = ['cloudflareError']
```

`let` has better (tighter) scoping semantics, which is the modern way of doing `var`.

Reply:
👍